pyproject.toml

```diff
@@ -47,6 +47,7 @@ pre-commit = ">=2.19.0"
 black = ">=22.6.0"
 isort = ">=5.10.1"
 openpyxl = ">=3.0.10"
+tables = ">=3.7.0"
```

Comment:
if we disable the missing imports check from pyright, we don't need this

Reply:
see comment above.  I don't think we should do this, because then  using `pandas-stubs` means you have to install `pytables`